apps/loan/src/components/PageLlamaMarkets/LendTableTitle.tsx

```diff
@@ -0,0 +1,66 @@
+import Box from '@mui/material/Box'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import LlamaSunglasses from '../../../public/images/llama-sunglasses.png'
+import Image from 'next/image'
+import styled from 'styled-components'
+import { t } from '@lingui/macro'
+import Typography from '@mui/material/Typography'
+import { InvertTheme } from '@ui-kit/shared/ui/ThemeProvider'
+
+const Img = styled(Image)`
+  flex-shrink: 1;
+  height: 276px;
+`
+
+const { MaxWidth, Sizing, Spacing } = SizesAndSpaces
+
+// note: The design seems to be using non-responsive values, this is not consistent!
+const { sm, md } = Spacing
+
+export const LendTableTitle = () => (
+  <Box sx={(t) => ({ backgroundColor: t.design.Layer[3].Fill, boxShadow: '0px 4px 4px -2px #2A334524' })}>
+    <Box maxWidth={MaxWidth.md} paddingBlock={Spacing.lg} display="flex" gap={sm} marginInline="auto">
+      <Img src={LlamaSunglasses} alt="Llama with sunglasses" />
+      <Box display="flex" flexDirection="column" gap={sm} flexGrow={1}>
+        <InvertTheme>
+          <Box
+            padding={Spacing.lg}
+            gap={Sizing.sm}
```

Comment:
I don't see no gap in the Figma

Reply:
yah there was a problem with the lineHeight being incorrect, fixed now